alembic/env.py

```diff
@@ -0,0 +1,80 @@
+from logging.config import fileConfig
+
+from sqlalchemy import engine_from_config
+from sqlalchemy import pool
+
+from alembic import context
+
+from db.database import Base
```

Comment:
Ensure that `Base` is correctly imported from `db.database` and that it contains the necessary metadata for Alembic to perform migrations. If `Base` is not defined or does not have the `metadata` attribute, this will cause issues during migration.